pkg/ingestor/parser/cyclonedx/parser_cyclonedx_test.go

```diff
@@ -52,7 +52,7 @@ func Test_cyclonedxParser(t *testing.T) {
 		wantPredicates: &testdata.CdxIngestionPredicates,
 		wantErr:        false,
 	}, {
-		name: "valid small CycloneDX document with package dependencies",
+		name: "valid small CycloneDX document with package dependencies and a hash",
```

Comment:
Based on this test, if the hash is provided for the top level, it should be used to create the `hasSBOM` node?

Reply:
nevermind I did not see the result change in the PR